assets/src/apps/authoring/components/PropertyEditor/PropertyEditor.tsx

```diff
@@ -14,6 +15,7 @@ interface PropertyEditorProps {
 
 const widgets = {
   CheckboxWidget: CustomCheckbox,
+  TextWidget: CustomTextBox,
```

Comment:
the built in TextWidget has a lot of features that I wonder if we're missing out on for this? https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/bootstrap-4/src/TextWidget/TextWidget.tsx

I wonder if it's really the text widget that is causing this or the change handler for the whole property editor. I also wonder if having this one field only save on blur while all the others are immediate might lead to some minor data loss issues (though I suppose it's not super likely that it'll keep focus until they close the browser, but I think that would be a scenario where it wouldn't save).